frontend/src/test/scala/bloop/BuildLoaderSpec.scala

```diff
@@ -247,6 +246,33 @@ object BuildLoaderSpec extends BaseSuite {
     }
   }
 
+  test("build respects different trace properties in separate workspaces") {
```

Comment:
This is a good test to make sure we don't make any mistake when loading settings from the workspace settings file. Re your previous question, I don't think it's necessary to add more tests than this, I'm happy with what we have